pkg/packagekit/updates.jsx

```diff
@@ -177,7 +177,8 @@ function HeaderBar(props) {
         if (!props.unregistered)
             actionButton = <button className="btn btn-default" onClick={props.onRefresh}>{_("Check for Updates")}</button>;
         if (props.timeSinceRefresh !== null)
-            lastChecked = cockpit.format(_("Last checked: $0 ago"), moment.duration(props.timeSinceRefresh * 1000).humanize());
+            lastChecked = cockpit.format(_("Last checked: $0"), moment(moment().valueOf() - props.timeSinceRefresh * 1000).locale(cockpit.language)
```

Comment:
`moment.locale()` should rather be called at the top of each page, once. Check `git grep moment.locale()` for existing usage.

Reply:
Ah, nice, was not aware of this :) Updated